src/sql/inspect.py

```diff
@@ -422,9 +422,14 @@ def __init__(self, table_name, schema=None) -> None:
             warning_background = "white"
             warning_title = ""
 
-        database = ConnectionManager.current.url
-        db_driver = ConnectionManager.current._get_database_information()["driver"]
-        if "duckdb" in database:
+        current = ConnectionManager.current
```

Comment:
I'm confused about this change, why is it needed?

also, you can access current.dialect, no need to add the `if` statement, check the developer docs for more details

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/785/files#diff-644b34a1c0db055c7b331734967239a598bd8f6e07cf50912c919bf70d243f3dR426)

Removed the if statement and changed to use `current.dialect` instead of `current.url` and `current._dialect`.

The change is there because: We need to distinguish whether a given connection is duckdb or not so that we can display an appropriate message that shows which statistics are not available. However, with `current.url` dbapi connection always returns None.